airbyte-webapp/src/hooks/services/Feature/constants.ts

```diff
@@ -1,10 +1,16 @@
 import { FeatureItem } from "./types";
 
-/** The default feature set that OSS releases should use. */
-export const defaultFeatures = [
+export const defaultOssFeatures = [
   FeatureItem.AllowCustomDBT,
   FeatureItem.AllowSync,
   FeatureItem.AllowUpdateConnectors,
   FeatureItem.AllowUploadCustomImage,
   FeatureItem.AllowSyncSubOneHourCronExpressions,
 ];
+
+export const defaultCloudFeatures = [
+  FeatureItem.AllowOAuthConnector,
+  FeatureItem.AllowSync,
+  FeatureItem.AllowChangeDataGeographies,
+  FeatureItem.AllowDBTCloudIntegration,
+];
```

Comment:
Perhaps unrelated to this PR, but I think this is more intuitive for our defaults for `FeatureService`.